src/config/apolloclient.ts

```diff
@@ -100,26 +103,18 @@ const gqlClient = (navigate: any) => {
       });
 
     if (networkError) {
-      // @ts-ignore
-      switch (networkError.statusCode) {
-        case 401:
-          setLogs('401 detected. Attempting token renewal before logout.', 'info', true);
-
-          // check if token exists
-          checkAndRenewToken(navigate);
-          break;
-        default:
-          // eslint-disable-next-line
-          Sentry.captureException(`[Network error]: ${networkError}`);
-          // logged error in logflare
-          setLogs(`[Network error]: ${networkError}`, 'error');
-      }
+      setLogs(`Network error: ${networkError} ${operation.variables}`, 'error');
+      navigate('/logout/session', { state: { to: window.location.pathname } });
```

Comment:
Does this include a temporary loss of internet? If so, then I am not sure if we should logout.